user_guide_src/source/incoming/controllers/010.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
-public function _remap()
+class myController extends BaseController
```

Comment:
`myController` is violation of naming convention in PSR-4 and CodeIgniter auto-routing spec.

```suggestion
class Products extends BaseController
```